### PR TITLE
Feature/license key copy

### PR DESCRIPTION
**Pull request by anthonyettinger**
Copy license key to clipboard

- medium pop size
- secondary action

@seanbrookes 
@jtary

### CODE REVIEW

**Comment by jtary** [comment]
LGTM

**Comment by anthonyettinger** [comment]
test please

File: client/www/scripts/modules/common/common.directives.js [hunk 1]
```diff
@@ -274,6 +278,7 @@ Common.directive('slPopoverInfo', [
         scope.icon = attrs.icon;
         scope.title = attrs.title || '';
         scope.hideOnPageClick = attrs.hideonpageclick;
+        $log.log('scope', scope);
```
Comment (seanbrookes) [inline client/www/scripts/modules/common/common.directives.js]: is this supposed to be here?

Comment (anthonyettinger) [inline client/www/scripts/modules/common/common.directives.js]: removed in next commit

**Comment by seanbrookes** [comment]
LGTM

**Pull request merged**